packages/opentelemetry-plugin-grpc-js/src/grpcJs.ts

```diff
@@ -14,32 +14,75 @@
  * limitations under the License.
  */
 
+import type * as grpcJs from '@grpc/grpc-js';
 import { BasePlugin } from '@opentelemetry/core';
+import * as shimmer from 'shimmer';
+import { patchClient, patchLoadPackageDefinition } from './client';
+import { patchServer } from './server';
 import { VERSION } from './version';
-import * as path from 'path';
-
-import * as grpcJs from '@grpc/grpc-js';
 
 /**
  * @grpc/grpc-js gRPC instrumentation plugin for Opentelemetry
  * https://www.npmjs.com/package/@grpc/grpc-js
  */
 export class GrpcJsPlugin extends BasePlugin<typeof grpcJs> {
   static readonly component = '@grpc/grpc-js';
+
   readonly supportedVersions = ['1.*'];
 
-  constructor(readonly moduleName: string, readonly version: string) {
+  tracer = this._tracer;
```

Comment:
why do you need to create here a public representation of its privates ? and this is also repeated in patch ?

Reply:
Since I've split this plugin's patching out into several files, typescript will complain about trying to access `protected` member `_tracer`. I could instead access via `plugin['_tracer']` constantly, but I thought this looked nicer.

I can add a comment to clarify this here, or access the private member via quotes instead. WDYT?

Reply:
what if you add public method `getTracer` and `getLogger` ?

Reply:
Updated in b7b2c93 to use public getters here instead.